apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -95,13 +72,15 @@ class OppgaveService(
                     throw BrukerManglerAttestantRolleException(saksbehandlerMedRoller.saksbehandler.ident)
                 }
             }
+
             is ExternalUser -> throw IllegalArgumentException("ExternalUser er ikke støttet for å tildele oppgave")
             else -> throw IllegalArgumentException(
                 "Ukjent brukertype ${appUser.name()} støtter ikke tildeling av oppgave",
             )
         }
     }
 
+    // TODO: Slå sammen tildel og bytt... Hvorfor er det to forskjellige?!!?!
```

Comment:
Burde ikke vi bare gjøre dette med en gang? :D

Reply:
Vil ikke ta for mye på én gang